### PR TITLE
Uniformly set FWHM to be pixels

### PR DESCRIPTION
**Pull request by rcooke-ast**
Proposal to set FWHM in pixels for both archive and object spectra.

### CODE REVIEW

File: pypeit/core/flexure.py [hunk 1]
```diff
@@ -668,23 +667,23 @@ def spec_flexure_slit(slits, slitord, slit_bpm, sky_file, method="boxcar", speco
             continue
 
         # get spectral FWHM (in Angstrom) if available
```
Comment (debora-pe) [inline pypeit/core/flexure.py]: please change this comment: `(in Angstrom)`--> `(in pixels)`, just to avoid confusion

**Review by debora-pe — APPROVED** [review]
It looks OK to me. It does look more clean and uniform like this. 
The only thing is that `wv_calib.wv_fits[iwv].cen_disp` and `obj_disp` are computed in a slightly different way, so it would be good to run the tests to check that everything looks still good.
Thanks.

**Comment by rcooke-ast** [comment]
Good idea @debora-pe -- nothing appears to be broken, and the results are consistent with what we were using before  :-)

**Old version:**
image

**New Version:**
image

File: pypeit/core/flexure.py (hunk 1 above)
Comment (rcooke-ast) [reply pypeit/core/flexure.py]: Good catch - updated!

**Review by profxj — APPROVED** [review]
thx for this!

**Comment by rcooke-ast** [comment]
Thanks for the approvals - I will merge #1572 first, and then merge this PR...

**Pull request merged**